admin/ops/update_product_metadata_schemas.py

```diff
@@ -0,0 +1,39 @@
+# type: ignore
+from dcpy.utils import s3
+from dcpy.models.product import metadata as product_metadata
+from dcpy.models.product.dataset import metadata_v2 as dataset_metadata
+
+import json
+from tempfile import TemporaryDirectory
+from pathlib import Path
+
+DO_SCHEMA_FOLDER = "data-engineering-devops/schemas/"
+
+schemas = [
+    {
+        "name": "org_metadata.schema.json",
+        "folder": "product/",
+        "schema": product_metadata.OrgMetadataFile.model_json_schema(),
+    },
+    {
+        "name": "product_metadata.schema.json",
+        "folder": "product/",
+        "schema": product_metadata.ProductMetadataFile.model_json_schema(),
+    },
+    {
+        "name": "dataset_metadata.schema.json",
+        "folder": "product/dataset/",
+        "schema": dataset_metadata.Metadata.model_json_schema(),
+    },
+]
+
+for schema in schemas:
+    with TemporaryDirectory() as _dir:
+        p = Path(_dir) / schema["name"]
+        open(p, "w").write(json.dumps(schema["schema"]))
+        s3.upload_file(
+            bucket="edm-publishing",
```

Comment:
There's an argument for a new bucket? More in that we could also have stuff specific to other buckets in here as well (ingest/library schemas, other devopsy stuff). But we can also just have those in their own buckets. Actually writing it out, I think this makes more sense. Carry on